simtools/data_model/data_reader.py

```diff
@@ -66,3 +68,62 @@ def read_table_from_file(file_name, schema_file=None, validate=False, metadata_f
         return _validator.validate_and_transform()
 
     return data_table
+
+
+def read_value_from_file(file_name, schema_file=None, validate=False):
```

Comment:
Not so important by I suggest adding this function to the __all__ list for consistency